pulseapi/entries/forms.py

```diff
@@ -39,3 +39,11 @@ def _save_m2m(self):
         for creator_profile_id in creator_profile_id_list:
             profile_id = int(creator_profile_id)
             EntryCreator.objects.create(entry=entry, profile_id=profile_id)
+
+    def clean_thumbnail(self):
```

Comment:
As code used here as well as `pulseapi/profiles/forms.py`, lets put this in its own file in the utility dir, and the import it in both these forms.py files? 

E.g. something like:
```
from pulseapi.utility.form_helpers import disallow_large_images

...

    def clean_thumbnail(self):
        data = self.cleaned_data.get("thumbnail", None)
        return disallow_large_images(data) 
```